osidb/models.py

```diff
@@ -559,6 +559,30 @@ class FlawMajorIncident(models.TextChoices):
 
         INVALID = "INVALID"
 
+    class FlawNistCvssValidation(models.TextChoices):
+        """
+        This flag determines the status of feedback loop between NIST and RH
+        requesting NVD CVSSv3 rescore for the CVE of this flaw.
+        The flag states have the following meaning:
+
+        * ( ) <unset>: No CVSSv3 rescore request was sent to NIST.
+        * (?) "requested": CVSSv3 rescore request was sent to NIST and the
+              feedback loop between NIST and RH is still opened. This status
+              is automatically set by SFM2 after sending request through Email
+              NIST functionality.
```

Comment:
Please remove `This status is automatically set by SFM2 after sending request through Email NIST functionality.`

Reply:
Oh. Fixed. Thanks.